index.html

```diff
@@ -5,8 +5,146 @@
+        </a>
+      <h3>Презентации</h3>
+      <p>Вы даже не подозреваете,  насколько вы изумительны!</p>
+      <a href="#">Заказать</a>
+      </li>
+    <ul>
+</section>
+<section>
+  <h2>О Компании</h2>
+<h3>Мы — маленькая, но гордая дизайн-студия из Краснодара.</h3>
+<p>Исповедуем принципы минимализма и чистоты. Ставим математический расчет превыше креатива. Работаем не покладая рук, как роботы.</p>
+<h4>Выполняем заказы на разработку:</h4>
+<ul>
+  <li>Веб-сайтов любой сложности</li>
+  <li>Мобильных приложений для iOS и Android</li>
+  <li>Слайдшоу и видео для корпоративных презентаций</li>
+</ul>
+  <a>
+    <img src="https://via.placeholder.com/  150x100" width="150" height="110" alt="Логотип «Нёрдс»">
+  </a>
+  <h4>с 2004 года Любим точность во всем:</h4>
+  <ul>
+    <li>
+      <p>146%<br>
+      Уровень самоотдачи</p>
+    </li>
+    <li>
+      <p>100%<br>
+      Соблюдение сроков</p>
+      <li>
+        <p>50%<br>
+        Минимальная предоплата</p>   
+  </ul>
+</section>
+<section>
+  <h2>Клиенты</h2>
+  <ul>
+   <li> <a>
+    <img src="https://via.placeholder.com/  150x100" width="150" height="110" alt="Логотип «HTML academy»">
+  </a></li>
+  <li> <a>
+    <img src="https://via.placeholder.com/  150x100" width="150" height="110" alt="Логотип «BORODINSKi»">
+  </a></li>
+  <li> <a>
+    <img src="https://via.placeholder.com/  150x100" width="150" height="110" alt="Логотип «PINK»">
+  </a></li>
+  <li> <a>
+    <img src="https://via.placeholder.com/  150x100" width="150" height="110" alt="Логотип «MISHKA»">
+  </a></li>
+</ul>
+</section>
+<section>
+</main>
+<footer>
+  <h2>Контакты</h2>
+  <p>
+    NЁRDS DESIGN STUDIO<br>
+    наб. Реки Карповки, 5 лит П, Санкт-Петербург<br>
+    Телефон: <a href="tel:+78005558628">+7 800 555-86-28</a>
+  </p>
+  <a href="blank.html">Напишите нам</a>
+  <!-- Здесь будет карта -->
+  </section>
+  </footer>
+  <section>
+    <ul>
+      <li>
+        <a href="#">Вконтакте</a>
+      </li>
+      <li>
+        <a href="#">Фейсбук</a>
+      </li>
+      <li>
+        <a href="#">Инстаграм</a>
+      </li>
+    </ul>
+    <h2>Давайте дружить, это выгодно!</h2>
+    <p>Скидка 10% для друзей из социальных сетей.</p>
+  </section>
+  <section>
+    <h2>Напишите нам</h2>
+    <form action="https://echo.htmlacademy.ru" method="post">
+      <p>
+        <label for="appointment-name">Ваше Имя</label>
+        <input id="appointment-name" type="text" name="name" placeholder="Имя Фамилия"> 
+      </p>
+      <p>
+      <label for="appointment-mail">Ваш email</label>
+        <input id="appointment-mail" type="mail" name="mail" placeholder="email@example.com">
+      </p>
+      <p>
+      <label for="appointment-appeal">Текст писmьа</label>
+      <textarea id="appointment-appeal"  name="appeal" placeholder="В свободной форме"></textarea>
+      </p>
+    <button type="button">Отправить</button>
```

Comment:
Внутри формы кнопка у нас будет отправлять то что пользователь заполнит на сервер, значит нам тут будет нужен тип кнопки `submit`

```suggestion
    <button type="submit">Отправить</button>
```